src/configs/all.ts

```diff
@@ -117,6 +117,7 @@ export const rules = {
     "no-use-before-declare": true,
     "no-var-keyword": true,
     "no-void-expression": true,
+    "prefer-conditional": true,
```

Comment:
How about `prefer-conditional-expression`? `if` and `? ... :` are both conditionals, but `? ... :` is an expression and `if` is a statement.